pkg/util/inittool/schema.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inittool
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/matrixorigin/matrixone/pkg/common/moerr"
+	"github.com/matrixorigin/matrixone/pkg/logutil"
+	"github.com/matrixorigin/matrixone/pkg/util/batchpipe"
+	"github.com/matrixorigin/matrixone/pkg/util/trace"
+	"strings"
+)
+
+type Column struct {
+	Name    string
+	Type    string
+	Default string
+	Comment string
+}
+
+var _ batchpipe.HasName = (*Table)(nil)
+
+type Table struct {
+	Database         string
+	Table            string
+	Columns          []Column
+	PrimaryKeyColumn []Column
+	Engine           string
+	Comment          string
+	CSVOptions       *trace.CsvOptions
+
+	BatchMode string
+}
+
+func (tbl *Table) GetName() string {
+	return tbl.Table
+}
+
+type optionsFactory func(db, tbl string) trace.TableOptions
+
+func (tbl *Table) ToCreateSql(ifNotExists bool, factory optionsFactory) string {
```

Comment:
Why do you need to write this?   Don't we already have this thing in show table command?

Reply:
`Table::ToCreateSql()` is for initialization, handle table creation.

key point is how to maintain diff metric data(have diff columns) export same table.
`Table` define all columns it has.
`View` declear `Table` and `Columns` which it depen. (PS: only for view initialization)
`Row` cooperate with `Table`, helps to output one line content (as `[]string`) for csv file.